web/packages/extension/assets/_locales/zh_TW/messages.json

```diff
@@ -56,7 +56,10 @@
     "settings_saved": {
         "message": "設定完成"
     },
-    "warn_on_unsupported_content": {
+    "settings_warn_on_unsupported_content": {
```

Comment:
Could you move these (`settings_warn_on_unsupported_content` and `settings_log_level`) back into alphabetical order?